third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -171,6 +221,15 @@ def to_dict(
         The type of the key-value pairs can be customized with the parameters
         (see below).
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> df = bpd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
```

Comment:
Can we add one example with user provided column names and show that the reflect after to_dict()?